### ISSUE TITLE
Repository : esc-backbone

### ISSUE THREAD

**Issue by l-wegner** [eclipse-tractusx/sig-infra#393]
Repo name: esc-backbone

Purpose: Implentation for open sourcing

Responsible: @Siegfriedk

**Comment by AngelikaWittek** [comment]
//awittek

**Comment by l-wegner** [comment]
Thank you @almadigabor , can you please also create a fork of that repo at catenax-ng? Do you need a separate ticket for that?

**Comment by almadigabor** [comment]
No, once the PR is merged and repo is created I can do the rest. It's already approved, who can I ping to merge it? https://github.com/eclipse-tractusx/.eclipsefdn/pull/45

**Comment by almadigabor** [comment]
Hey @l-wegner! The repo is available so I created the fork. You can reach it [here](https://github.com/catenax-ng/tx-esc-backbone).

**Comment by almadigabor** [comment]
@l-wegner I've moved this issue to in review, let me know if I can close it.

**Comment by FaGru3n** [comment]
Hi @l-wegner any updates or can we close this issue?

Thanks in advance.

**Comment by FaGru3n** [comment]
Hi @l-wegner if you need further support please let us know, meanwhile i would close the issue.
